frontend/src/scenes/feature-flags/FeatureFlags.tsx

```diff
@@ -24,13 +25,24 @@ export const scene: SceneExport = {
     logic: featureFlagsLogic,
 }
 
+const searchFeatureFlags = (sources: FeatureFlagType[], search: string): FeatureFlagType[] => {
+    return new Fuse(sources, {
+        keys: ['key', 'name'],
+        threshold: 0.3,
+    })
+        .search(search)
+        .map((result) => result.item)
+}
+
 export function FeatureFlags(): JSX.Element {
     const { currentTeamId } = useValues(teamLogic)
     const { featureFlags, featureFlagsLoading } = useValues(featureFlagsLogic)
     const { updateFeatureFlag, loadFeatureFlags } = useActions(featureFlagsLogic)
     const { push } = useActions(router)
     const { tableScrollX } = useIsTableScrolling('lg')
 
+    const [searchTerm, setSearchTerm] = useState(false as string | false)
+
```

Comment:
... but I'm going to ask changes here. Can you move all the new stuff inside `featureFlagsLogic`? We still want to: 1) keep all our data in logics, and 2) not want to redo the search on every render.

Reply:
Done